content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -112,7 +112,7 @@ The Corefile configuration includes the following [plugins](https://coredns.io/p
 * [kubernetes](https://coredns.io/plugins/kubernetes/): CoreDNS will reply to DNS queries
   based on IP of the Services and Pods. You can find [more details](https://coredns.io/plugins/kubernetes/)
   about this plugin on the CoreDNS website.
-  - `ttl` allows you to set a custom TTL for responses. The default is 5 seconds.
+  - `ttl` allows you to set a custom TTL for responses. The default is 30 seconds.
```

Comment:
Thanks, but the default CoreDNS TTL is not 30 seconds.

Try this:
```suggestion
  - `ttl` allows you to set a custom time to live (TTL) for responses. The example above
    configures CoreDNS to cache records for up to 30 seconds, even if it receives a
    record with a longer TTL.
```

Reply:
> Thanks, but the default CoreDNS TTL is not 30 seconds.

I know; sorry for shortening the PR title too much. I meant _default TTL when deployed through kubeadm_ (ref: https://github.com/search?q=repo%3Akubernetes%2Fkubernetes%20%2Fttl%20%5B%5Cd%5D%2F&type=code)

---

But then I completely misunderstood the current wording. My bad.

I was trying to figure out the default TTL inside a K8S cluster and thought the snippet and the text got out of sync.

---

How about `The default configuration above [...]`?